db/migrate/201607221959_add_request_help_to_user_exercise.rb

```diff
@@ -0,0 +1,5 @@
+class AddRequestHelpToUserExercise < ActiveRecord::Migration
+  def change
+    add_column :user_exercises, :request_help, :boolean, default: false
```

Comment:
I like boolean fields to read as a question that you can answer yes or no to. E.g. "active". Yes, active; no, inactive. Here "request_help" is phrased as a command, which makes it more confusing when it's a boolean.